pkg/master/master.go

```diff
@@ -182,8 +182,10 @@ type Config struct {
 	// allow downstream consumers to disable swagger
 	EnableSwaggerSupport bool
 	// allow api versions to be conditionally disabled
-	DisableV1 bool
-	EnableExp bool
+	DisableV1                 bool
+	EnableExtensions          bool
+	EnableExtensionResources  map[string]struct{} // If present, only these resources are enabled in the extensions api group, all others are disabled. This is ignored if EnableExtensions is false.
```

Comment:
Sorry for not being clear-- I meant combining them. I'm pretty sure it's
got some odd behavior right now if you both specifically enable and disable
some resources.

On Mon, Oct 12, 2015 at 6:15 PM, Nikhil Jindal <notifications@github.com>
wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/15516#discussion_r41818286>
> :
>
> > @@ -182,8 +182,10 @@ type Config struct {
> >  	// allow downstream consumers to disable swagger
> >  	EnableSwaggerSupport bool
> >  	// allow api versions to be conditionally disabled
> > -	DisableV1 bool
> > -	EnableExp bool
> > +	DisableV1                 bool
> > +	EnableExtensions          bool
> > +	EnableExtensionResources  map[string]struct{} // If present, only these resources are enabled in the extensions api group, all others are disabled. This is ignored if EnableExtensions is false.
>
> Are you suggesting combining Enable and Disable maps or just changing each
> of them to map[string]bool?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15516/files#r41818286>.
>